tracing-journald/tests/journal.rs

```diff
@@ -0,0 +1,98 @@
+#![cfg(target_os = "linux")]
+
+use std::collections::HashMap;
+use std::process::Command;
+
+use tracing::{error, info, warn};
+use tracing_journald::Subscriber;
+use tracing_subscriber::subscribe::CollectExt;
+use tracing_subscriber::Registry;
+
+fn journalctl_version() -> std::io::Result<String> {
+    let output = Command::new("journalctl").arg("--version").output()?;
+    Ok(String::from_utf8_lossy(&output.stdout).to_string())
+}
+
+fn with_journald(f: impl FnOnce()) {
+    match journalctl_version() {
+        Ok(_) => {
+            let sub = Registry::default().with(Subscriber::new().unwrap().with_field_prefix(None));
+            tracing::collect::with_default(sub, f);
+        }
+        Err(error) => eprintln!(
+            "SKIPPING TEST: journalctl --version failed with error: {}",
+            error
+        ),
```

Comment:
:+1: this seems good to me. note that this won't be printed to the console by default, though, because of libtest's output capturing.

Reply:
I know, but at least I can pass `--nocapture` to see what's going on. Otherwise I'd have no idea 🙂